### DISCUSSION TITLE
Persistent Messages show 0 in the UI

### DISCUSSION THREAD

**Discussion by lancepowell** [rabbitmq/rabbitmq-server#3395] [Other]
UI Bug: Persistent Messages show as 0, however messages are persistent and the message body bytes show correctly.

This is on 3.9.5.

![image (8)](https://user-images.githubusercontent.com/1454879/132698980-9f3d1909-6c1b-441c-a422-73758e7f6ca6.png)

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
We don’t have any information to work with here, sorry. Steps to reproduce and evidence of persistent messages, please. We don’t even know what your topology and publisher code looks like.

**Comment by lancepowell** [comment]
Sure, I just assumed you would notice it pretty quickly in testing. We run
docker containers and upgraded to 3.9.5, we have 30ish clusters spanning
across all of our infrastructure. All of these handle only persistent
messages. I've done manual validation to ensure the messages are persisted,
and they are which is good. It's just odd in the UI you see the bytes are
persisted but the number of messages is always 0. Let me know if you want
any more details, happy to help out.

**Reply by lancepowell** [reply]
Steps to reproduce, created a durable classic queue and published a persistent message. The UI only shows bytes are persisted and not the number of messages.

**Reply by lancepowell** [reply]
sorry multiple replies, the screen shot is taken from the queue page in the management UI.
